### PR TITLE
feat: ComboBox

### PR DESCRIPTION
**Pull request by MaximeDion-Work**
GitHub Issue: #

## PR Type

What kind of change does this PR introduce?

- Feature

## PR Checklist 
Please check if your PR fulfills the following requirements:

- Commits must be following the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [x] Tested UWP
- [x] Tested iOS
- [ ] Tested Android
- [ ] Tested WASM
- [x] Contains **No** breaking changes
  > If the pull request contains breaking changes, commit message must contain a detailed description of the action to take for the consumer of this library. As explained by the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)

### CODE REVIEW

File: src/library/Uno.Material/Styles/Controls/ComboBox.xaml [hunk 1]
```diff
@@ -0,0 +1,429 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+					xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+					xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:xamarin="http://nventive.com/xamarin"
+					xmlns:toolkit="using:Uno.UI.Toolkit"
+					mc:Ignorable="xamarin">
+
+	<!-- System Resources (Comment/Uncomment/Add/Remove resources you need)-->
```
Comment (MatFillion) [inline src/library/Uno.Material/Styles/Controls/ComboBox.xaml:9]: what does this comment mean? It's in a library, no one will edit this :)

File: src/library/Uno.Material/Styles/Controls/ComboBox.xaml [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+					 Opacity="0.6" />
+	<SolidColorBrush x:Key="ComboBoxForegroundThemeBrush"
+					 Color="{ThemeResource OnSecondaryColor}" />
+	<SolidColorBrush x:Key="ComboBoxFocusedBackgroundThemeBrush"
+					 Color="{ThemeResource OnSecondaryColor}"
+					 Opacity="0.6" />
+
+	<SolidColorBrush x:Key="ComboBoxPlaceholderTextForegroundThemeBrush"
+					 Color="{ThemeResource OnSecondaryColor}"
+					 Opacity="0.6" />
```
Comment (MatFillion) [inline src/library/Uno.Material/Styles/Controls/ComboBox.xaml:44]: all those colors should point to the proper brush directly, not recode the opacity.

Like this` <StaticResource x:Key="TextBoxLabelDefaultColorBrush" ResourceKey="OnSurfaceMediumBrush" />`

**Comment by MatFillion** [comment]
I checked out the branch to test and it looks good. One detail though, there's a uno option to make it open always below. I would use that I think.

To avoid this. (it's default in windows, but not in material I think?)
@JessicaNventive  What do you think.
image

File: Uno.Material.sln [hunk 3]
```diff
@@ -30,13 +30,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "workflows", "workflows", "{
 	EndProjectSection
 EndProject
 Global
-	GlobalSection(SharedMSBuildProjectFiles) = preSolution
```
Comment (MatFillion) [inline Uno.Material.sln:33]: that's a weird change ?

Comment (MaximeDion-Work) [reply Uno.Material.sln:33]: Oh yeah I don't know where this is coming from.

**Pull request merged**